src/app/pages/Swap/form/SwapFormInput/SwapSelectTokenFace.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC } from 'react';
+
+import { Button, IconBase } from 'app/atoms';
+import { ReactComponent as CompactDown } from 'app/icons/base/compact_down.svg';
+import { TezosTokenIconWithNetwork } from 'app/templates/AssetIcon';
+import { setTestID } from 'lib/analytics';
+import { T } from 'lib/i18n';
+
+interface SwapFieldProps {
+  tezosChainId: string;
+  assetSlug?: string;
+  assetSymbol: string;
+  onSelectAssetClick: EmptyFn;
+  testId?: string;
+}
+
+const SwapSelectTokenFace: FC<SwapFieldProps> = ({
+  tezosChainId,
+  assetSlug,
+  assetSymbol,
+  onSelectAssetClick,
+  testId
+}) => (
+  <div {...setTestID(testId)}>
+    {assetSlug ? (
+      <Button
+        onClick={onSelectAssetClick}
+        className="bg-white py-[3px] px-2 rounded-8 flex items-center justify-between -mr-2.5 cursor-pointer w-[120px]"
+      >
+        <div className="flex items-center">
+          <TezosTokenIconWithNetwork tezosChainId={tezosChainId} assetSlug={assetSlug} size={32} />
+          <span className="text-text text-xs font-semibold overflow-hidden max-w-10 text-ellipsis">{assetSymbol}</span>
+        </div>
+        <IconBase Icon={CompactDown} className="text-primary" size={16} />
```

Comment:
```suggestion
        <IconBase Icon={CompactDown} className="text-primary"/>
```
16 is the default size